_pages/getting-started/classes/benefits.md

```diff
@@ -210,6 +210,64 @@ Note that the gross income in your paycheck is a little less than your annual in
 
+## Public Service Loan Forgiveness (PSLF)
+
+Public Service Loan Forgiveness (PSLF) is a government program that offers forgiveness of federal student loans under certain conditions. Further information on the program and whether you qualify for PSLF can be found on [the PSLF webpage](https://studentaid.gov/manage-loans/forgiveness-cancellation/public-service).
+
+As a GSA employee, you are eligible for PSLF (assuming you satisfy the other conditions). 
+
+To fill out your PSLF application and obtain employer verification:
+
+1. Download the application PDF from the [PSLF application page](https://studentaid.gov/manage-loans/forgiveness-cancellation/public-service/public-service-loan-forgiveness-application)
+1. Fill out your Section 1 of the application with your personal information.
+1. Check the appropriate box in Section 2 depending on whether you think you currently qualify for PSLF or not. 
+1. Add your signature to the bottom of page 1. **IMPORTANT:** You can sign the document in pen/ink or
+using a digital signature. But if you use a digital signature, **it must be hand-drawn**. Adobe Reader provides the ability [to sign documents with hand-drawn signatures](https://helpx.adobe.com/reader/using/sign-pdfs.html).
+1. Enter the following information in Section 3:
+    - **Employer Name:** General Services Administration
+    - **Federal Employer Identification Number (FEIN):** 44-0553234
+    - **Employer Address:** 1800 F St. NW, Washington, D.C. 20405
+    - **Employer Website:** www.gsa.gov
+    - **Employment Begin Date:** Whenever your posting officially began. You can find this date in [HRLinks](https://hrlinks.gsa.gov) under `Employee Personal Info -> View/Update Personal Info -> Personal Information -> Employee Information`.
+    - **Employment End Date:** Check "Still Employed"
+    - **Employment Status:** Check "Full-Time"
+    - **Hours Per Week (Average):** 40
+    - **Is your employer a governmental organization?:** Check "Yes"
+1. **Do not fill out Section 4**. This section will be completed by a GSA HR representative as detailed below.
+1. Save your updated PDF.
+1. Login to [DocuSign](https://sign.gsa.gov).
+1. Click "Start" and then "Send an Envelope".
+1. Under `Add Documents to the Envelope`, click the "Upload" button and select your partially completed PSLF application. Once the document is uploaded, you should a pop-up message like "We found form fields for public-service-application-for-forgiveness.pdf."
+
+    ![Screenshot of DocuSign interface to upload a document with pop-up that says "We found form fields for public-service-application-for-forgiveness.pdf."]({{site.baseurl}}/images/pslf/pslf-upload-document.png)
+
+1. Under `Add Recipients to the Envelope`, add Vivian Carrasco, vivian.carrasco@gsa.gov as the recipient of the envelope
+1. On the recipient box for Vivian, click `Customize -> Advanced Settings`
```

Comment:
I don’t know if we have a hard rule about this, but pages with specific names tend to get outdated quickly as folks move around the organization (although selfishly I hope Vivi never leaves!). You may want to reference [our benefits contact](https://handbook.tts.gsa.gov/general-information-and-resources/employee-resources-policies/benefits-links/#benefits-and-retirement-related-questions) instead of the name here.

Reply:
@adunkman Good point. In this case, I added her information directly based on her input in a previous Slack conversation. However, would it be better to say that people should use the "Workforce Relations HR Specialist" listed in this document? Since in this case it would direct people to the same person.

https://docs.google.com/document/d/15glvq9UakKUN8XTRTa6gRkhBHm2whhQyAGmf8ibTtBs/edit